pkg/providers/azure.go

```diff
@@ -25,42 +24,39 @@ type SetUpCmd struct {
 	Provider          string
 	Repo              string
 	appId             string
-	tenantId          string
+	TenantId          string
 	appObjectId       string
 	spObjectId        string
-	AzClient          AzClient
+	AzClient          AzClientInterface
 }
 
-func InitiateAzureOIDCFlow(ctx context.Context, sc *SetUpCmd, s spinner.Spinner) error {
+const AZURE_CONTRIBUTOR_ROLE_ID = "b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
Lets make this:

```
const AKS_ADMIN_ROLE_ID = "3498e952-d568-435e-9b2c-8d77e338d7f7"
```

to limit scope

https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#containers

Reply:
This will match the portal/vscode experiences as well